src/testing/docker/test-standalone.sh

```diff
@@ -19,9 +20,9 @@ readonly HOST="$(docker port "${CONTAINER_ID}" 80)"
 curl --silent --location "http://${HOST}/repo/" | grep -q "<title>Getting Started with FOSSology</title>"
 
 #### test copyright is running
-docker exec -it "${CONTAINER_ID}" /usr/local/share/fossology/copyright/agent/copyright -h
+docker-compose exec -T web /usr/local/share/fossology/copyright/agent/copyright -h
```

Comment:
The script is made to test the Docker environment built from single container and not from `docker-compose`. For that, the file `src/testing/docker/test-cluster.sh` is used. Can you please revert the changes to `docker exec`?